shared/src/androidMain/kotlin/com/kcc/kmmhackathon/shared/utility/SharedDate.kt

```diff
@@ -1,5 +1,15 @@
 package com.kcc.kmmhackathon.shared.utility
 
+import java.time.Instant
+import java.time.LocalDateTime
+import java.time.ZoneId
+import java.time.format.DateTimeFormatter
 import java.util.*
 
-actual typealias SharedDate = Date
\ No newline at end of file
+actual typealias SharedDate = Date
+
+fun timeInMillisToReadableString(timeInMillis: Long): String {
+    val formatter = DateTimeFormatter.ofPattern("EEE dd MMM YYYY HH:mm")
```

Comment:
Found that EEE is the way to get the shorter DayOfWeek in formatter.

Reply:
I haven't tested but it will be great if we could use `dateStyle` and `timeStyle` with maybe with `DateTimeFormatter.ofLocalizedDateTime`